roles/swift-common/templates/etc/sudoers.d/swiftops

```diff
@@ -1,3 +1,7 @@
-swiftops ALL= NOPASSWD: /usr/local/bin/swifttool
+{% set basepath = openstack_source.virtualenv_base if
+openstack_install_method == 'source' else
+openstack_package.virtualenv_base %}
+
+swiftops ALL= NOPASSWD: {{ basepath }}/swift/bin/swifttool
```

Comment:
Instead of changing the path to execution, why not use the alternatives system to place swiftool into the normal path?